authenticating-users/app.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright 2019, Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const express = require('express');
+const got = require('got')
+const jwt = require('jsonwebtoken');
+
+const app = express();
+
+// Cache externally fetched information for future invocations
+let CERTS;
+let AUDIENCE;
+
+async function certs() {
+  if (!CERTS) {
+    let response = await got('https://www.gstatic.com/iap/verify/public_key');
+    CERTS = JSON.parse(response.body)
```

Comment:
I need to update the linter config to run on this. There's a few `;` missing.

Reply:
That's what I get for switching languages often. I think I caught and fixed them all.

Reply:
I think all are fixed now.